src/mbgl/map/transform.cpp

```diff
@@ -46,6 +46,10 @@ Transform::Transform(MapObserver& observer_,
 #pragma mark - Map View
 
 bool Transform::resize(const Size size) {
+    if (!size) {
+        return false;
+    }
```

Comment:
Because if we allow `Transform` to have a zero-ed size, it can result in division-by-zero issues e.g. [here](https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/map/transform_state.cpp#L45).